src/styles/blocks/calendar.scss

```
@@ -0,0 +1,38 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: 706px;
+  padding: $indent;
+  @extend %day-font;
+
+  @each $day, $shift in $week {
+    &--start-day--#{$day} *:first-child {
+      margin-left: $shift;
+    }
+  }
+
+  @each $none-day in 28, 29, 30, 31 {
+    &--month-length--#{$none-day} *:not(:nth-child(-n + #{$none-day})) {
+      display: none;
+    }
+  }
+
+  &__block {
+    box-sizing: border-box;
+    @include square($block-size);
+    transition: $transition-time;
```

You should specify transition properties directly
Default is all, but it could affect old browsers' performance 
```suggestion
    transition: background-color $transition-time, transform $transition-time;
```